vendor/vendor.json

```diff
@@ -3,595 +3,595 @@
 	"ignore": "test",
 	"package": [
 		{
-			"checksumSHA1": "9NR0rrcAT5J76C5xMS4AVksS9o0=",
```

Comment:
Please leave this file out of your commit. I don't know why all checksums are different, but they shouldn't be.